src/exprs.h

```diff
@@ -61,22 +84,31 @@
 **  Note that 'EVAL_EXPR' does not use 'TNUM_EXPR', since it also handles the
 **  two special cases that 'TNUM_EXPR' handles.
 */
-#define EVAL_EXPR(expr) \
-                        (IS_REFLVAR(expr) ? OBJ_REFLVAR(expr) : \
-                         (IS_INTEXPR(expr) ? OBJ_INTEXPR(expr) : \
-                          (*EvalExprFuncs[ TNUM_STAT(expr) ])( expr ) ))
+
+static inline Obj EVAL_EXPR(Expr expr)
+{
+    if (IS_REFLVAR(expr)) {
+        return OBJ_REFLVAR(expr);
+    }
+    else if (IS_INTEXPR(expr)) {
+        return OBJ_INTEXPR(expr);
+    }
+    else {
+        return (*EvalExprFuncs[TNUM_STAT(expr)])(expr);
+    }
+}
 
 
 /****************************************************************************
 **
-*V  EvalExprFuncs[<type>]  . . . . . evaluator for expressions of type <type>
```

Comment:
This bit looks a little strange as I had to rearrange the order of a couple of things, now we are using functions rather than macros.